src/inverseRobot.js

```diff
@@ -24,7 +24,17 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const newRobot = {};
+
+  for (const key in robot) {
+    newRobot[robot[key]] = key;
+  }
+
+  if (Object.keys(robot).length !== Object.keys(newRobot).length) {
+    return null;
+  }
```

Comment:
You don't need this, check if `newRobot` has key you're trying to add in previous loop